src/TestableIO.System.IO.Abstractions.TestingHelpers/IMockFileDataAccessor.cs

```diff
@@ -86,8 +87,40 @@ public interface IMockFileDataAccessor : IFileSystem
         PathVerifier PathVerifier { get; }
 
         /// <summary>
-        /// Gets a reference to the underlying file system. 
+        /// Gets a reference to the underlying file system.
         /// </summary>
         IFileSystem FileSystem { get; }
+
+        /// <summary>
+        /// Add a new file that is empty.
+        /// </summary>
+        /// <param name="path">A string representing the path of the new file to add.</param>
+        void AddEmptyFile(string path);
+
+        /// <summary>
+        /// Add a new file that is empty.
+        /// </summary>
+        /// <param name="path">An <see cref="IFileInfo"/> representing the path of the new file to add.</param>
+        void AddEmptyFile(IFileInfo path);
+
+        /// <summary>
+        /// Add a new, empty directory.
+        /// </summary>
+        /// <param name="path">An <see cref="IDirectoryInfo"/> representing the path of the new directory to add.</param>
+        void AddDirectory(IDirectoryInfo path);
+
+        /// <summary>
+        /// Add a new file with its contents set to a specified <see cref="MockFileData"/>.
+        /// </summary>
+        /// <param name="path">An <see cref="IFileInfo"/> representing the path of the new file to add.</param>
+        /// <param name="data">The data to use for the contents of the new file.</param>
+        void AddFile(IFileInfo path, MockFileData data);
+
+        /// <summary>
+        /// Gets a file.
+        /// </summary>
+        /// <param name="path">The path of the file to get.</param>
+        /// <returns>The file. <see langword="null"/> if the file does not exist.</returns>
+        MockFileData GetFile(IFileInfo path);
```

Comment:
I don' think these new methods need to be here. This interface is mostly used by other `Mock*` classes. (Actually it should probably be `internal`, but that would be a severe breaking change :/)

Reply:
I have removed all changes to this file.